src/main/java/seedu/address/logic/commands/ListStudentCommand.java

```diff
@@ -12,12 +22,69 @@
 
     public static final String COMMAND_WORD = "list_student";
 
-    public static final String MESSAGE_SUCCESS = "Listed all students";
+    public static final String MESSAGE_SUCCESS_ALL = "Listed all students";
+
+    public static final String MESSAGE_SUCCESS = "Listed students from class: %1$s";
```

Comment:
```suggestion
    public static final String MESSAGE_SUCCESS_CLASS = "Listed students from class: %1$s";
```

Reply:
not necessary, but may be more intuitive for the reader to know that the purpose of 2 different success messages

Reply:
Thanks! Will make the changes.